Copied from upstream repository.
Hunk #2 is removed since it cannot be applied and it is not needed.

From 7cead7137bf54851c1b7df5a3854351296d21276 Mon Sep 17 00:00:00 2001
From: Vasantha Ganesh K <vasanthaganesh.k@tuta.io>
Date: Thu, 22 Jun 2017 23:38:40 +0530
Subject: [PATCH] removed `check-conventions' from make

---
 Makefile                          |   7 +-
 tests/haskell-code-conventions.el | 165 ------------------------------
 2 files changed, 1 insertion(+), 171 deletions(-)
 delete mode 100644 tests/haskell-code-conventions.el

diff --git a/Makefile b/Makefile
index b2c89d6..aa907c5 100644
--- a/Makefile
+++ b/Makefile
@@ -79,12 +79,7 @@ build-$(EMACS_VERSION)/build-flag : build-$(EMACS_VERSION) $(patsubst %.el,build
 check-%: tests/%-tests.el
 	$(BATCH) -l "$<" -f ert-run-tests-batch-and-exit;
 
-check: compile $(AUTOLOADS) check-ert check-conventions
-
-check-conventions :
-	$(BATCH) -l tests/haskell-code-conventions.el                                           \
-                 -f haskell-check-conventions-batch-and-exit
-	@echo "conventions are okay"
+check: compile $(AUTOLOADS) check-ert
 
 check-ert: $(ELCHECKS)
 	$(BATCH) --eval "(when (= emacs-major-version 24)					\
-- 
2.18.0

